.env.development

```diff
@@ -1,4 +1,5 @@
 LOAD_DEV_SUITES=*
 VALIDATOR_URL=https://inferno.healthit.gov/validatorapi
 JS_HOST=http://localhost:3000
-BASE_PATH=inferno
\ No newline at end of file
+BASE_PATH=inferno
+FHIR_ENDPOINT_URL=https://inferno.healthit.gov
```

Comment:
I think `FHIR_ENDPOINT_URL` is too generic, and this should be something like `REFERENCE_SERVER_URL`.